# https://github.com/microsoft/react-native-test-app/pull/1934
diff --git a/test-app.gradle b/test-app.gradle
index b516bf86d9d8dbf6ccb497ea261c10827cd8eac7..39bbedc1413ebaabe04509efb4e754636f715121 100644
--- a/test-app.gradle
+++ b/test-app.gradle
@@ -67,9 +67,12 @@ ext.applyTestAppSettings = { DefaultSettings settings ->
     settings.project(":support")
         .projectDir = file("${testAppDir}/android/support")

+    def reactNativeVersion = getPackageVersionNumber("react-native", rootDir)
+
     def reactNativeGradlePlugin =
-        findNodeModulesPath("@react-native/gradle-plugin", reactNativeDir)  // >= 0.72
-            ?: findNodeModulesPath("react-native-gradle-plugin", reactNativeDir)  // < 0.72
+        reactNativeVersion >= v(0, 72, 0)
+            ? findNodeModulesPath("@react-native/gradle-plugin", reactNativeDir)
+            : findNodeModulesPath("react-native-gradle-plugin", reactNativeDir)
     if (reactNativeGradlePlugin != null) {
         settings.includeBuild(reactNativeGradlePlugin)
     }
